tests/csm/rest/test_query_deployment.py

```diff
@@ -0,0 +1,112 @@
+        cls.ha_obj = HAK8s()
+        cls.deploy_lc_obj = ProvDeployK8sCortxLib()
+        cls.csm_obj = csm_api_factory("rest")
+        cls.csm_conf = configmanager.get_config_wrapper(fpath="config/csm/test_rest_capacity.yaml")
+        cls.update_seconds = cls.csm_conf["update_seconds"]
+
+    def setup_method(self):
+        """
+        Setup method
+        """
+        self.log.info("Prerequisite: Deploy cortx cluster")
+        self.log.info("Cleanup: Destroying the cluster ")
+        resp = self.deploy_lc_obj.destroy_setup(self.csm_obj.master, self.csm_obj.worker_list,
+                                                K8S_SCRIPTS_PATH)
+        assert_utils.assert_true(resp[0], resp[1])
+        self.log.info("Cleanup: Cluster destroyed successfully")
+
+        self.log.info("Cleanup: Setting prerequisite")
+        self.deploy_lc_obj.execute_prereq_cortx(self.csm_obj.master,
+                                                K8S_SCRIPTS_PATH,
+                                                K8S_PRE_DISK)
+
+        for node in self.csm_obj.worker_list:
+            self.deploy_lc_obj.execute_prereq_cortx(node, K8S_SCRIPTS_PATH,
+                                                    K8S_PRE_DISK)
+        self.log.info("Cleanup: Prerequisite set successfully")
+
+        self.log.info("Cleanup: Deploying the Cluster")
+        resp_cls = self.deploy_lc_obj.deploy_cluster(self.csm_obj.master,
+                                                     K8S_SCRIPTS_PATH)
+        assert_utils.assert_true(resp_cls[0], resp_cls[1])
+        self.log.info("Cleanup: Cluster deployment successfully")
+
+        self.log.info("[Start] Sleep %s", self.update_seconds)
+        time.sleep(self.update_seconds)
+        self.log.info("[Start] Sleep %s", self.update_seconds)
+
+        self.log.info("Cleanup: Check cluster status")
+        resp = self.ha_obj.poll_cluster_status(self.csm_obj.master)
+        assert_utils.assert_true(resp[0], resp[1])
+        self.log.info("Cleanup: Cluster status checked successfully")
+
+    @pytest.mark.lc
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-45672')
+    def test_45675(self):
+        """
+        Verify GET cluster topology with valid storage id
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        self.log.info("Open and parse solution yaml")
+        self.log.info("Step 1: Send GET request for fetching system topology"
+                      "without storage set id")
+        resp, result = self.csm_obj.verify_storage_set()
+        assert result, "Verification for get system topology failed."
+        self.log.info("Response : %s", resp)
+        self.log.info("Step 2: Send GET request for fetching system topology"
+                      "with storage set id")
+        storage_sets = resp.json()["topology"]["cluster"]["storage_set"]
```

Comment:
["cluster"] is the list. U will cluster ID to extract the storage set
So plan to to loop for all the clusters

Reply:
done